src/odbc_connection.cpp

```diff
@@ -4,18 +4,24 @@
 
 namespace odbc {
 
-void odbc_connection::set_current_result(odbc_result* r) {
-  if (r == current_result_) {
+void odbc_connection::cancel_current_result(bool quiet) {
```

Comment:
My C++ is extremely rusty, but I think this is simple enough that I didn't mess it up. (Famous last words!)